airlock/templates/file_browser/group.html

```diff
@@ -32,17 +32,15 @@
         {% fragment as comment_status %}
           <span>
             {% if request.user.output_checker %}
-              {% if release_request.get_turn_phase.name == "INDEPENDENT" %}
+              {% if release_request.get_turn_phase.name == "INDEPENDENT" and comment.review_turn == release_request.review_turn %}
```

Comment:
Should this logic be in the view somewhere?

Reply:
Yes, but I was trying to avoid doing too much extra reworking in this PR. There's a lot in this template that's still doing logic things that should be in the view.

I can move this one, though. Probably best not to add more.

Reply:
Actually, this is per-comment logic, so I think I'll leave it for a later PR.

Reply:
yep, that does make it awkward